test/extended/cluster/cl.go

```diff
@@ -79,6 +79,9 @@ var _ = g.Describe("[Feature:Performance][Serial][Slow] Load cluster", func() {
 				err := oc.Run("new-project").Args(nsName).Execute()
 				o.Expect(err).NotTo(o.HaveOccurred())
 				e2e.Logf("%d/%d : Created new namespace: %v", j+1, p.Number, nsName)
+				// label namespace nsName
+				_, err = SetNamespaceLabels(c, nsName, p.Labels)
```

Comment:
If you check the other `Create{Configmaps,Secrets}` calls we like to check if the object exists before making the function call. I see that you have added the check in the `SetNamespaceLabels`, but could we (also?) have the check in line here (as below)?